editoast/Dockerfile

```diff
@@ -81,6 +81,16 @@ RUN --mount=type=cache,target=/usr/local/cargo/registry \
 FROM alpine:3.20 AS running_env
 RUN apk add --no-cache jemalloc curl ca-certificates geos libpq openssl postgresql16-client
 
+# install `fga` binary used to compile the authorization model at runtime
+RUN mkdir install \
+    && cd install \
+    && wget https://github.com/openfga/cli/releases/download/v0.6.4/fga_0.6.4_linux_amd64.tar.gz \
+    && tar xf fga_0.6.4_linux_amd64.tar.gz \
+    && mv fga /usr/local/bin \
+    && cd .. \
+    && rm -rf install
```

Comment:
We should try to install a CLI compatible with the architecture. We could try `go install github.com/openfga/cli/cmd/fga@latest` and then copy the built binary to different images.